src/styles/main.scss

```diff
@@ -0,0 +1,69 @@
+$day-size: 100px;
+$border-size: 1px;
+$border-color: #000;
+$gap-size: 1px;
+$padding-size: 10px;
+$day-bg: #eee;
+$hover-bg: #ffbfcb;
+$font-size: 30px;
+$font-family: Arial, sans-serif;
+$animation-duration: 0.5s;
+
+html,
+body {
+  height: 100%;
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  padding: $padding-size;
+  width: calc(7 * #{$day-size} + 6 * #{$gap-size} + 2 * #{$padding-size});
```

Comment:
Consider using a variable for the number of days in a week (7) to avoid magic numbers and make the code more readable. For example, you could define a variable $days-in-week: 7; and then use it in this calculation.